helm/postgres/templates/postgres.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     {{ .Values.user }}:  # database owner
     - superuser
     - createdb
+    {{- range .Values.extraUsers }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
   resources:
   {{- mergeOverwrite .Values.default.resources .Values.resources | toYaml | nindent 4 }}
   databases:
```

Comment:
We'd also want the option to add databases and set a user as the owner of that database.

Reply:
Done